bundles/specmate-migration/src/com/specmate/migration/h2/SQLUtil.java

```diff
@@ -44,6 +45,23 @@ public static void executeStatements(List<String> queries, Connection connection
 		}
 	}
 	
+	public static int getIntResult(String query, int resultIndex, Connection connection) throws SpecmateException {
```

Comment:
There is no closing of the statement or the result set. Is this necessary?

Reply:
Yes, good catch! Fixed.